src/pages/workspaces/workspace/Data.js

```diff
@@ -545,9 +552,13 @@ const EntitiesContent = ajaxCaller(class EntitiesContent extends Component {
                         ])
                       ]),
                       cellRenderer: ({ rowIndex }) => {
-                        return renderDataCell(
-                          Utils.entityAttributeText(entities[rowIndex].attributes[name]), namespace
-                        )
+                        const dataInfo = entities[rowIndex].attributes[name]
+                        return h(Fragment, [
+                          (dataInfo && _.isArray(dataInfo.items)) ? linkButton({
+                            onClick: () => this.setState({ viewData: dataInfo })
+                          },
+                          [renderDataCell(Utils.entityAttributeText(dataInfo), namespace)]) : renderDataCell(Utils.entityAttributeText(dataInfo), namespace)
```

Comment:
Nit: It would be good to find a way to avoid duplicating `renderDataCell(Utils.entityAttributeText(dataInfo), namespace)`... either extracting a `const` or passing it to a function that conditionally wraps it in a `linkButton`.